packages/core/src/__tests__/__helpers__/mockSegmentStore.ts

```diff
@@ -16,15 +16,15 @@ import type {
 import { createCallbackManager } from './utils';
 import { createGetter } from '../../storage/helpers';
 
-type Data = {
+export type StoreData = {
   isReady: boolean;
   context?: DeepPartial<Context>;
   settings: SegmentAPIIntegrations;
   userInfo: UserInfoState;
   deepLinkData: DeepLinkData;
 };
 
-const INITIAL_VALUES: Data = {
+const INITIAL_VALUES: StoreData = {
   isReady: true,
```

Comment:
@oscb Shouldn't `isReady` initially be `false`?

Reply:
I need to remove these, we don't use them anymore (we have more granular readiness checks now). Good catch.